python/skupper_router/management/skrouter.json

```diff
@@ -137,7 +137,14 @@
             "description": "The standard AMQP management node interface.",
             "fullName": true,
             "extends": "operationalEntity",
-            "operations": ["QUERY", "GET-TYPES", "GET-ANNOTATIONS", "GET-OPERATIONS", "GET-ATTRIBUTES", "GET-MGMT-NODES"],
+            "operations": [
```

Comment:
whoops, did not know that clang-format will want to format jsons; means that json has to be added to the filter in git pre-commit, otherwise it's going to create nasty surprise

Reply:
https://github.com/skupperproject/skupper-router/pull/669